src/Actions.js

```diff
@@ -85,7 +100,61 @@ function joinListserv(firstName, lastName, callback) {
     } else {
       gapi.auth2.getAuthInstance().signIn().then(() => {listServSubscribe()})
     }
+}
+
+//check if the user is blacklisted when they try to sign up for the ListServ and 
+// again for the GroupMe
+async function isBlacklisted(user_id, id_type) {
+  
+  var userData = {
+    'userId': user_id,
+    'idType': id_type
+  };
+  // **
+  ////need to remember to change the api URI after testing
+  // **
+  var resp = await fetch("http://localhost:3001" + "/isblacklisted",
+    {
+    method: 'POST',
+    body: JSON.stringify(userData)
+    }
+  ) 
+  
+  if (resp.ok){
+    console.log('Success');
+    //**double check this below when we actually have the server. I had to 
+    //change it slightly when i used AWS Lambda, idk why.
+    var blacklisted = await resp.json();
+    console.log(blacklisted);
+    return blacklisted;
+  } else {
+    alert("HTTP-Error: " + resp.status)
+  };  
+}
+
+async function addEvent(goog_id, goog_token, gm_token) {
+  var eventData = {
+    'userId': goog_id,
+    'googleAccessToken': goog_token, 
+    'gmToken': gm_token
+  }
+  // **
+  ////need to remember to change the api URI after testing
+  // **
+  var resp = await fetch("http://localhost:3001" + "/addevent",
+    {
+    method: 'POST',
+    body: JSON.stringify(eventData)
+    }
+  );
+  
+  if (resp.ok) {
+    console.log('Success');
+  } else {
+    alert("HTTP-Error:" + resp.status);
   }
+}
+
 
 export {
     joinGroupMe,
```

Comment:
This file did not need to see any changes, probably should just revert this back to what master has